otx/algorithms/detection/adapters/mmdet/config_utils.py

```diff
@@ -272,7 +274,7 @@ def set_data_classes(config: Config, labels: List[LabelEntity]):
     for subset in ("train", "val", "test"):
         cfg = get_data_cfg(config, subset)
         cfg.labels = labels
-        config.data[subset].labels = labels
+        #  config.data[subset].labels = labels
```

Comment:
This should be deleted but I left it commented for a reference purpose.
If a dataset is a wrapper dataset such as `RepeatDataset`, `MultiImageMixDataset`, etc., not `OTXDataset`, assigning `labels` config in them does not make sense.
For `RepeatDataset` and others, it is okay to set it safely, since they are not used when initializing but it is not the case for `MultiImageMixDataset` because all the config will be used to initialize it.
Please refer these lines in `build_dataset`: https://github.com/open-mmlab/mmdetection/blob/master/mmdet/datasets/builder.py#L74-L77
Note that only `MultiImageMixDataset` use all the config presented but not other wrapper dataests

Reply:
@harimkang Could you give some advice on this?

Reply:
I think it won't matter. As long as this doesn't affect other training, we can delete it. Alignment is required for Dataset Wrapping configuration. This can also be true for segmentation.
@jaegukhyun What do you think?

Reply:
Let's see the test results later. Especially on the YOLOX side.